composer.json

```diff
@@ -26,21 +26,23 @@
 		"issues": "https://github.com/infinum/eightshift-forms/issues",
 		"source": "https://github.com/infinum/eightshift-forms"
 	},
+
 	"require-dev": {
 		"brain/faker": "dev-master",
 		"brain/monkey": "^2.6.1",
-		"dealerdirect/phpcodesniffer-composer-installer": "^0.7.2",
-		"infinum/eightshift-coding-standards": "^1.6",
+		"dealerdirect/phpcodesniffer-composer-installer": "^v1.0.0",
+		"infinum/eightshift-coding-standards": "2.0.0-beta",
+		"php-parallel-lint/php-parallel-lint": "^1.3",
```

Comment:
Should we hold this off until the issue with non-escaped folder names has been resolved?
https://github.com/php-parallel-lint/PHP-Parallel-Lint/pull/171#issuecomment-2049605273

Reply:
I will check what I can do on my side, but we will need to address this definitely :/